.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="SquareAndroid" />
```

Comment:
this line should not be here, you may have to go to your settings and delete the codestyle "SquareAndroid"

Reply:
I ran `git rm -r --cached . ` to try and fix these .idea files not being 'gitignored' ([commit](https://github.com/kiwix/kiwix-android/pull/1276/commits/bdf979d973130af01ee48a55128ea5795fe03871)), guess I messed up

Reply:
no, I have added our codestyle to git(and our inspections) so nobody has to "install" the codestyle anymore, completely automatic

Reply:
[Done](https://github.com/kiwix/kiwix-android/pull/1276/commits/35a0bffb478e279758c4096d8ced4ea571e3771e)